crypto/src/main/java/org/cryptimeleon/incentive/crypto/model/keys/user/UserKeyPair.java

```diff
@@ -1,13 +1,18 @@
 package org.cryptimeleon.incentive.crypto.model.keys.user;
 
+import lombok.AllArgsConstructor;
 import lombok.Value;
-import org.cryptimeleon.math.serialization.annotations.Represented;
+import org.cryptimeleon.craco.sig.sps.eq.SPSEQSignature;
 
 @Value
+@AllArgsConstructor
 public class UserKeyPair {
-    @Represented
```

Comment:
why removed?

Reply:
Did not do anything, the class was not representable. I don't know why we added it in the first place :smile: